Dockerfile.processing_image

```diff
@@ -21,7 +21,7 @@ RUN git clone https://github.com/chanzuckerberg/single-cell-explorer.git \
 # Install python dependencies
 # Remove packaging dependency once pyparser>3 is supported.
 RUN pip3 install scanpy==1.6.0 python-igraph==0.8.3 louvain==0.7.0 packaging==21.0 awscli
-RUN python3 -m pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple/ cellxgene-schema==2.1.0rc0
+RUN pip3 install cellxgene-schema==2.1.0
```

Comment:
Replacing the Test PyPI version of `cellxgene-schema` with the one released on PyPI.

Reply:
`cellxgene-schema` 2.1.0 was released to PyPI this morning:
* https://pypi.org/project/cellxgene-schema/2.1.0/